model/project.go

```diff
@@ -750,6 +750,9 @@ func (p *Project) FindTaskForVariant(task, variant string) *BuildVariantTaskUnit
 	if bv == nil {
 		return nil
 	}
+	if bv.IsInDisplayTask(task) {
```

Comment:
Should this warn?

Reply:
Probably not. The point is that execution tasks don't exist from the user's perspective, so silently ending seems like the right call.